services/Mail.js

```diff
@@ -0,0 +1,45 @@
+import dotenv from 'dotenv';
+import sgMail from '@sendgrid/mail';
+import emailMesseges from './mailMessage';
+
+dotenv.config();
+sgMail.setApiKey(process.env.SENDGRID_API_KEY);
+
+/**
+ * @exports Mail
+ * @class AuthController
+ * @description Handles the user verification mail sending
+ *
+ */
+class Mail {
+  /**
+   * @static
+   * @param {object} req
+   * @param {object} res
+   * @param {function} next
+   * @return {json} Returns json object
+   * @memberof Mail
+   */
+  static sendVerification(req, res, next) {
+    const { email, token } = req.user;
+    const url = `http://localhost:3000/api/v1/auth/confirmation/${token}`;
+    const msg = emailMesseges.signupVerification(email, url);
```

Comment:
You might want to change  "emailMesseges" to "emailMessages"

Reply:
Thanks, I will change it now